boards/rp-pico/examples/pico_hd44780_display.rs

```diff
@@ -0,0 +1,137 @@
+//! # LCD Display Example
+//!
+//! In this example, the RP2040 is configured to drive a small two-line
+//! alphanumeric LCD using the
+//! [HD44780](https://crates.io/crates/hd44780-driver) driver.
+//!
+//! It drives the LCD by pushing data out of six GPIO pins. This version has
+//! been adapted from rp-hal/pr2040-hal/examples/lcd_display.rs for 
+//! the rp-pico board layout and pin assignment.
+//!
+//! See the `Cargo.toml` file for Copyright and license details.
+//!
+//! ```text              /--------------------------------------\
```

Comment:
Just a detail: In the generated docs, this line is not shown, but put into the `class` attribute, probably because it's in the same line as the block start marker:
```
<div class="example-wrap"><pre class="language-text              /--------------------------------------\"><code> ____________        |        /-------------------------\   |
| 1       GND|-------+---\    |           _|USB|_       |   |
| 2       VDD|-------+---+----/          |1  R 40|-VBUS-o   v
| 3        VS|-------/   |               |2  P 39|       ||POT||
| 4        RS|--\        o-----------GND-|3    38|-GND----------o
| 5        RW|--+--------/    /------GP2-|4  P 37|
```